openshift/rh-che.app.yaml

```diff
@@ -149,8 +149,10 @@ objects:
             timeoutSeconds: 60
           resources:
             limits:
+              cpu: 1000m
```

Comment:
Do we have data on whether this limit will be sufficient? `1000m` seems low to me as a maximum.

Reply:
Should be more than enough for the last week the usage was around 0.05  - also see http://file.rdu.redhat.com/jmelisba/report-201910211441.html